node-zerox/src/utils.ts

```diff
@@ -11,9 +11,18 @@ export const encodeImageToBase64 = async (imagePath: string) => {
 
 // Strip out the ```markdown wrapper
 export const formatMarkdown = (text: string) => {
-  const formattedMarkdown = text
-    .replace(/^```[a-z]*\n([\s\S]*?)\n```$/g, "$1")
-    .replace(/^```\n([\s\S]*?)\n```$/g, "$1");
+  let formattedMarkdown = text;
```

Comment:
We might need to trim the text, just in case (sometimes the responses contain spaces at the end/beginning)
```
  let formattedMarkdown = text?.trim();
```